eng/configure-toolset.sh

```diff
@@ -19,6 +19,8 @@ function Test-FilesUseTelemetryOutput {
         'eng/common/native/install-cmake-test.sh'
         'eng/common/native/common-library.sh'
         'eng/common/native/install-cmake.sh'
+        'eng/common/cross/arm/tizen-fetch.sh'
```

Comment:
@Haplois Does it makes sense to have that as part of the translation PR?

Reply:
It doesn't :( I pushed it to the wrong branch